src/main/java/screens/user_registration_screen/UserRegistrationUI.java

```diff
@@ -80,17 +86,24 @@ public void getUserCredentials(){
     }
 
     public static void main(String[] args){
-        //Testing purposes
-        Database testDB = new UserDatabase(new File("test301"));
-        UserExistsInputBoundary interactor = new UserExistsInteractor(testDB);
-        new UserRegistrationUI(interactor).getUserCredentials();
+        Database testDB = new UserDatabase(new File("newAccounts23"));
+        UserLoginInteractor2 userLoginInteractor2 = new UserLoginInteractor2(testDB, new UserChatsPresenter());
```

Comment:
Also here, I'm not completely sure, but I think the file path would have to be the same in all places of our code so whenever we create a new database it can access the same file

Reply:
This was just for trying out the UI, once we log in tho, it will no longer work, as other components use a different database.